website/docs/brain/brain-next-steps.md

```diff
@@ -0,0 +1,20 @@
+---
+id: next-steps
+title: Next Steps
+---
+
+## 1. Examples
+
+Navigate to the `Examples` tab, or go directly to the [Examples Index](./../examples/examples_index.md),
```

Comment:
i think that all the internal relative links you can somehow pass the page `id` so that i t doesn't break in case we move things around

Reply:
OK we can try next docs update